OPERATING.md

```diff
@@ -0,0 +1,163 @@
+# Operating Guide: Data Transformation and Ingest for VEDA
+
+This guide provides information on how VEDA runs data ingest, transformation and metadata (STAC) publication workflows via AWS Services, such as step functions.
+
+## Data
+
+### `collections/`
+
+The `collections/` directory holds json files representing the data for VEDA collection metadata (STAC).
+
+Should follow the following format:
+
+```json
+{
+    "id": "<collection-id>",
+    "type": "Collection",
+    "links":[
+    ],
+    "title":"<collection-title>",
+    "extent":{
+        "spatial":{
+            "bbox":[
+                [
+                    "<min-longitude>",
+                    "<min-latitude>",
+                    "<max-longitude>",
+                    "<max-latitude>",
+                ]
+            ]
+        },
+        "temporal":{
+            "interval":[
+                [
+                    "<start-date>",
+                    "<end-date>",
+                ]
+            ]
+        }
+    },
+    "license":"MIT",
+    "description": "<collection-description>",
+    "stac_version": "1.0.0",
+    "dashboard:is_periodic": "<true/false>",
+    "dashboard:time_density": "<month/>day/year>",
+    "item_assets": {
+        "cog_default": {
+            "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+            "roles": [
+                "data",
+                "layer"
+            ],
+            "title": "Default COG Layer",
+            "description": "Cloud optimized default layer to display on map"
+        }
+    }
+}
+
+```
+
+### `events/`
+
+The `events/` directory holds json files representing the step function inputs for initiating the discovery, ingest and publication workflows.
+Can either be a single event or a list of events.
+
+Should follow the following format:
+
+```json
+{
+    "collection": "<collection-id>",
+    "discovery": "<s3/cmr>",
+
+    ## for s3 discovery
+    "prefix": "<s3-key-prefix>",
+    "bucket": "<s3-bucket>",
+    "filename_regex": "<filename-regex>",
+    "datetime_range": "<month/day/year>",
```

Comment:
@abarciauskas-bgse `datetime_range` is specified in the input to the step function